h2/src/test/org/h2/test/TestAll.java

```diff
@@ -431,6 +432,7 @@
     /** If not null the database should be opened with the collation parameter */
     public String collation;
 
+    public boolean stdDropTableRestrict;
 
```

Comment:
rahter than adding this boolean, just pass it in the getConnection() call inside testDropTable